common/render_shaders.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 // UnicodeCap is the amount of unicode characters the fonts will be able to use, starting from index 0.
-var UnicodeCap = 200
+const UnicodeCap = 200
```

Comment:
This one shouldn't be `const`, because users should be able to edit this (i.e. it needs to be set to 1900 for arabic, but because of the performance cost, it's defaulted to 200)